syntax = "proto3";

package org.signal.registration.rpc;

option swift_prefix = "";

service RegistrationService {
    rpc ValidateCredentials (ValidateCredentialsRequest) returns (ValidateCredentialsResponse);
}

message ValidateCredentialsRequest {
    string user_id = 1;
    string password = 2;
}

message ValidateCredentialsResponse {
    string phone_number = 1;
}
